keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourcePersister.kt

```diff
@@ -132,10 +132,8 @@ class ResourcePersister(
     }
 
   private fun DeliveryArtifact.register() {
-    if (!artifactRepository.isRegistered(name, type)) {
```

Comment:
Always re-register to allow changes to docker versioning strategy.

Reply:
Maybe only re-register if the version strategy has actually changed, then? Or at least only publish the event when that is the case?

Reply:
That seems like an unneeded optimization - what would it save us?

Reply:
It would avoid polluting the event log with duplicate events? Or is this event marked with the automatic non-dupe thing?

Reply:
It's not recorded in the event log